storage/innobase/handler/ha_innodb.cc

```diff
@@ -19054,6 +19057,25 @@ wsrep_innobase_kill_one_trx(
 	DBUG_RETURN(0);
 }
 
+static
+void
+wsrep_wait_until_initialized(
+/*====================*/
+       handlerton*)
+{
+	extern bool trx_rollback_is_active;
+	DBUG_ENTER("wsrep_innodb_wait_untill_initialized");
+
+	WSREP_DEBUG("waiitng until innodb is initialized %d",
+		   trx_rollback_is_active);
+	while (trx_rollback_is_active) {
```

Comment:
This does not work for wsrep.binlog_format, wsrep.mdev_6832 and wsrep.mdev_7798 test cases, all cases it hangs here.